lib/sles4sap/sap_deployment_automation_framework/configure_tfvars.pm

```diff
@@ -14,6 +14,8 @@ use Carp qw(croak);
 use utils qw(file_content_replace);
 use sles4sap::sap_deployment_automation_framework::deployment qw(get_os_variable);
 use sles4sap::sap_deployment_automation_framework::deployment_connector qw(find_deployment_id);
+use Mojo::Base qw(publiccloud::basetest publiccloud::azure);
```

Comment:
This does not set basetest, only imports functions, right?

Reply:
No, not imports functions but set basetest.

Reply:
Hmm... ok I see. But id does not look like causing any troubles so far. 
I guess that might answer question from @mpagot above. We can't set basetest to PC lib at test module level as it would break stuff.